src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -5004,6 +5004,12 @@ int wm_vuldet_collect_agent_software(scan_agent *agent, sqlite3 *db, scan_ctx_t*
                 architecture = agent->arch;
             }
 
+            char* win_architecture = NULL;
+            if (agent->dist == FEED_WIN && architecture) {
+                os_strdup(architecture, win_architecture);
```

Comment:
This will not be required with the suggestion that I made for the `wm_vuldet_normalize_architecture_NVD` method.

Reply:
Fixed!